### PR TITLE
Feature/front end1

### PR DESCRIPTION
**Pull request by avishkar58**
This pull request contains the following changes:
* Swapped react-bootstrap with Bootstrap 4
* Added a skeleton login form
* Added a skeleton "create account" form

### CODE REVIEW

**Comment by KaleabTessera** [comment]
This looks good!

I will make minor changes to this tonight:
1. Create a separate Login Component and import it into the login page.
2. Try and create a template or consistent method for service calls.

File: api/README.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ $ docker-compose run web nosetests
 
 Migrations for the provided models are part of the seed project. To generate new migrations use `Flask-Migrate`:
 
-    $ docker-compose run web python run.py db migrate --directory api/migrations
-    $ docker-compose run web python run.py db upgrade --directory api/migrations
+    $ docker-compose run web python ./api/run.py db migrate --directory api/migrations
+    $ docker-compose run web python ./api/run.py db upgrade --directory api/migrations
```
Comment (jaderabbit) [inline api/README.md]: thank you! missed that

**Comment by avishkar58** [comment]
> This looks good!
> 
> I will make minor changes to this tonight:
> 
> 1. Create a separate Login Component and import it into the login page.
> 2. Try and create a template or consistent method for service calls.

Cool - thanks! :)

**Comment by KaleabTessera** [comment]
Things done:
- So I have taken the LoginForm and CreateAccountForm out of their pages and put them in separate components.
- Updated readme with new adding code instructions.

Todo:
- Update the service calls to use [axios](https://alligator.io/react/axios-react/). It handles service calls better than the standard fetch api. I will complete this tomorrow and merge this pull req in.

**Comment by avishkar58** [comment]
Thanks for making those changes!

**Pull request merged**